### PR TITLE
JW-383: Moved if_* functions to if_windows.c

### PR DESCRIPTION
**Pull request by sodar**

### CODE REVIEW

File: utils/if_windows.c [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    return -1;
+}
+
+// TODO: JW-351
+char *
+if_indextoname(unsigned int ifindex, char *ifname)
+{
+    printf("%s: Not supported on Windows\n", __func__);
+    return NULL;
+}
```
Comment (michal-clapinski) [inline utils/if_windows.c]: new line

File: utils/vif.c [hunk 2]
```diff
@@ -27,6 +27,9 @@
 #include <windows.h>
 #include "wingetopt.h"
 #include "stdbool.h"
+
+extern unsigned int if_nametoindex(const char *ifname);
+extern char *if_indextoname(unsigned int ifindex, char *ifname);
```
Comment (michal-clapinski) [inline utils/vif.c]: what was wrong with vr_utils_temp.h?

Comment (michal-clapinski) [inline utils/vif.c]: i mean why did you move this?

Comment (sodar) [inline utils/vif.c]: Those definitions in `vr_utils_temp.h` trigger an error during Agent build.

```
J:\workspace\build-vrouter---4cd830f6@2\vrouter\include\vr_utils_temp.h(54): error C2732: linkage specification contradicts earlier specification for 'if_nametoindex'
```

Comment (sodar) [inline utils/vif.c]: Even with `extern "C" { ... }` around it.
